public/css/main.css

```diff
@@ -120,6 +120,108 @@ body {
 
 /*** ADD EVENT FORM ***/
 
+.add-event-form {
+  max-width: 100vw;
+  padding: 4rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.add-event-form > label {
+  font: bold 1.8rem 'Lato';
+  color: #5C5C5C;
+}
+
+
+.form-input {
+  height: 3rem;
+  width: 100%;
+  border: 0.1rem gray solid;
+  border-radius: 0.2rem;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+  background-color: white;
+}
+
+.event-description {
+  height: 5rem;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+  border-radius: 0.2rem;
+  border: 0.1rem gray solid;
+}
+
+.checkbox-wrap {
+  padding: 1rem;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+  border: 0.1rem gray solid;
+  flex-flow: wrap;
+  display: flex;
+  border-radius: 0.2rem;
+}
+
+.checkbox-wrap label {
+  width: 50%;
+  font: bold 1.4rem 'Lato';
+  margin-top: 0.2rem;
+  margin-bottom: 0.2rem;
+  color: #5C5C5C;
+}
+
+.checkbox-wrap input {
+  width: 15%;
+}
+
+.place-section {
+  flex-direction: column;
+  display: flex;
+  width: 100%;
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
+}
+
+.place-wrap {
+  margin-top: 0.5rem;
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.place-wrap label {
+  font: 1.8rem 'Lato';
```

Comment:
this should be the same as the others - you've got multiple classes for all the labels when you only really need one

Reply:
Also the plus icon needs to be the same color as the label text - you can just add a custom class (I see you used a font awesome one to change the size)

Reply:
what about the checkbox labels they need to be different font size, so i cant just give every label the same styling